src/DropDownMenu/DropDownMenu.js

```diff
@@ -241,7 +261,7 @@ class DropDownMenu extends Component {
 
   handleTouchTapControl = (event) => {
     event.preventDefault();
-    if (!this.props.disabled) {
+    if (!this.props.disabled && !(this.props.multiple && this.state.open)) {
```

Comment:
From my own tests, that callback isn't called when the dropdown is touched. I think that it's a dead logic.